requirements.txt

```diff
@@ -3,7 +3,7 @@ pytest>=6.2.5
 coverage==7.1.0
 flake8==6.0.0
 google-cloud-bigquery~=3.7.0
-PyYAML~=5.4
+PyYAML~=5.3.1
```

Comment:
Perhaps adding `>=` instead of `~=` would be better here? Does it work for you in this case?

Reply:
Indeed, that worked fine, updated accordingly.